intelmq/bots/parsers/shadowserver/README.md

```diff
@@ -7,3 +7,60 @@ This module is maintained by [The Shadowserver Foundation](https://www.shadowser
 
 Please contact intelmq@shadowserver.org with any issues or concerns.
 
+The report configuration is now stored in a _shadowserver-schema.json_ file downloaded from https://interchange.shadowserver.org/intelmq/v1/schema.
+
+The parser will attempt to download a schema update on startup when the *auto_update* option is enabled.
+
+Schema downloads can also be scheduled as a cron job:
+
+```
+02  01 *   *   *     intelmq.bots.parsers.shadowserver.parser --update-schema
+```
+
+For air-gapped systems automation will be required to download and copy the file to VAR_STATE_PATH/shadowserver-schema.json.
+
+The parser will automatically reload the configuration when the file changes.
+
+
+## Schema contract
+
+Once set the `classification.identifier`, `classification.taxonomy`, and `classification.type` fields will remain static.
+
+Once set report fields will not be deleted.
```

Comment:
I don't understand these sentences. Do you mean
> Once set (by the user?), the `classification.identifier`, `classification.taxonomy`, and `classification.type` fields will remain static.

or something else. Better be more verbose

Reply:
Updated in the documentation.  We won't change the classifications which would break event filtering for users.